client/client.go

```diff
@@ -32,8 +32,7 @@ import (
 // Status responses are returned in the result structure, and can be cast
 // to built-in error instance (or in the returned error if the client is
 // configured accordingly). Certain statuses can be checked using `apistatus`
-// and standard `errors` packages. Note that package provides some helper
-// functions to work with status returns (e.g. IsErrContainerNotFound).
+// and standard `errors` packages.
```

Comment:
BTW, using `[errors]` can be much better than `errors`.

Reply:
It looks better in godoc, done.